Reformat API Ref links in "Overview of APIs", clarify Stable in Release/Prerelease (#2555)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/concepts/overview-features-apis.md

<!-- maintenance notes: add table rows for any new h2 sections -->

> This page only lists Release APIs; it doesn't list Experimental APIs or Prerelease-stable APIs (that is, APIs which reached Stable status in a Prerelease SDK, but haven't yet been included in a Release SDK).  For Experimental APIs and Prerelease-stable APIs, see [Release Notes for the WebView2 SDK](../release-notes.md).

Author: @novac42 @peiche-jessica @nishitha-burman - pls review status names

Reviewer: ```suggestion
This page only lists APIs in Release SDKs; it doesn't list Experimental APIs or Stable APIs which are yet available in Prerelease SDKs.  For a comprehensive list of APIs including Experimental, see [Release Notes for the WebView2 SDK](../release-notes.md).
```
I'm more inclined to only describing APIs with "stable" and "experimental", and SDKs/packages with "release" and "prerelease".

Author: Using @peiche-jessica's wording, except have to change from:
Stable APIs which are yet available in Prerelease SDKs.
to either of:
Stable APIs which are not yet available in Release SDKs <--
Stable APIs which are still only available in Prerelease SDKs
so gives:
This page only lists APIs in Release SDKs; it doesn't list Experimental APIs, or Stable APIs which are not yet available in Release SDKs.  For a comprehensive list of APIs including Experimental APIs, see [Release Notes for the WebView2 SDK](../release-notes.md).

Author: adding hidden comment to the new table of 3 status rows, like:
try to only describe APIs as "stable" and "experimental", and only describe SDKs/packages as "release" and "prerelease"
I will try out such terminology usage in the edited spots, rather than creating a status called "Release".

   * [CoreWebView2.IsSuspended Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.issuspended)
   * [CoreWebView2.Resume Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.resume)
   * [CoreWebView2.OpenTaskManagerWindow Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.opentaskmanagerwindow) 

##### [WinRT/C#](#tab/winrtcsharp)

* `CoreWebView2` Class:
   * [CoreWebView2.MemoryUsageTargetLevel Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#memoryusagetargetlevel)
   * [CoreWebView2.TrySuspendAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#trysuspendasync)
   * [CoreWebView2.IsSuspended Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#issuspended)
   * [CoreWebView2.Resume Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#resume)
   * [CoreWebView2.OpenTaskManagerWindow Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#opentaskmanagerwindow)

##### [Win32/C++](#tab/win32cpp)

> * `ICoreWebView2Experimental5` interface:<!--todo: delete "experimental" links?-->

Author: @peiche-jessica @novac42 @nishitha-burman - How did an Experimental (prerelease) item get into the Overview of APIs article?  keep or delete this get/put property, in Win32 tab?  In .NET & WinRT tabs?  Is there a non-Experimental version of this getter/setter?  

What will be the non-Experimental version of this getter/setter?  Guessing:
ICoreWebView2_123::get_MemoryUsageTargetLevel
Search:
https://learn.microsoft.com/search/?scope=WebView2&terms=get_MemoryUsageTargetLevel
Find "win32"

Author: Removing from the 3 tabs.
